Sources/SwiftDates/Date+Formatting.swift

```diff
@@ -6,7 +6,8 @@ import Foundation
 public extension Date {
     /// Returns an ISO 8601 formatter set to locale
     /// Thanks, Daniel Marques, https://github.com/erica/SwiftDates/issues/2
-    public static var iso8601Formatter: ISO8601DateFormatter = {
+    @available(OSX 10.12, *)
+    static var iso8601Formatter: ISO8601DateFormatter = {
```

Comment:
Xcode automatically removed the public modifier, because it's redundant. It should be removed from the other properties as well. If you want me to, I can do that, while i'm at it.